osu.Game.Tests/Visual/Gameplay/TestSceneStarCounter.cs

```diff
@@ -49,6 +49,7 @@ public TestSceneStarCounter()
         private void setStars(float stars)
         {
             starCounter.Current = stars;
+            starCounter.Current = starCounter.Current > 10 ? starCounter.Current = 10 : starCounter.Current = starCounter.Current;
```

Comment:
- i'm not sure why this is being done at all in a test scene of all places
- it reads very weird too, because in both branches of this ternary `starCounter.Current` is being assigned the same value _twice_. based on this i'm not sure if you understand how ternary expressions work - they're not just different syntax for `if...else` blocks, and i'd recommend researching the difference if you don't know it

Reply:
If starCounter.Current is > 10, assign 10 else keep the same value? 

_(that's what I was going for)_

Reply:
yeah but the ternary itself returns a value, which is then assigned to the left-hand side of the `=` operator. you don't need to assign a *second* time inside the ternary - that's redundant and confusing.

Reply:
> * i'm not sure why this is being done at all in a test scene of all places
> * it reads very weird too, because in both branches of this ternary `starCounter.Current` is being assigned the same value _twice_. based on this i'm not sure if you understand how ternary expressions work - they're not just different syntax for `if...else` blocks, and i'd recommend researching the difference if you don't know it

I've also updated this ~